frontend/tests/unit/models/team-test.js

```diff
@@ -0,0 +1,13 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+
+module('Unit | Model | team', function(hooks) {
+  setupTest(hooks);
+
+  // Replace this with your real tests.
+  test('it exists', function(assert) {
+    let store = this.owner.lookup('service:store');
+    let model = store.createRecord('team', {});
+    assert.ok(model);
+  });
```

Comment:
If there's no real logic inside the models to test, I'd say we just delete those tests. I see that I didn't do that myself so its fine you left them in but I'd say to just delete them if they have no purpose